### PR TITLE
Refactor testenv.yaml and remove TestConfiguration class

### PR DESCRIPTION
**Pull request by krdln**
cc @sodar @m-kostrzewa 

This is a preparation to enable agent service tests.

### CODE REVIEW

File: CIScripts/Test/TestConfigurationUtils.ps1 [hunk 1]
```diff
@@ -322,83 +328,77 @@ function Initialize-TestConfiguration {
...
-            -AdapterName $TestConfiguration.AdapterName
+            -AdapterName $SystemConfig.AdapterName
 
     # Prepare parameters for script block
-    $ControllerIP = $TestConfiguration.ControllerIP
+    $ControllerIP = $ControllerConfig.Address
     $VHostIfName = $HNSTransparentAdapter.ifName
     $VHostIfIndex = $HNSTransparentAdapter.ifIndex
 
+    # TODO ???
```
Comment (katrybacka) [inline CIScripts/Test/TestConfigurationUtils.ps1]: More explicit explanation could be a better hint for reader.

File: CIScripts/Test/TestRunner.ps1 [hunk 2]
```diff
@@ -6,13 +6,11 @@ function Invoke-TestScenarios {
     Param (
         [Parameter(Mandatory = $true)] [PSSessionT[]] $Sessions,
         [Parameter(Mandatory = $true)] [String] $TestenvConfFile,
-        [Parameter(Mandatory = $true)] [String] $TestConfigurationFile,
         [Parameter(Mandatory = $true)] [String] $TestReportOutputDirectory
     )
 
     $TestsBlacklist = @(
         # Put filenames of blacklisted tests here.
-        "vRouterAgentService.Tests.ps1"
```
Comment (katrybacka) [inline CIScripts/Test/TestRunner.ps1]: Are you sure that this test should be enabled in this PR?

File: CIScripts/Test/Tests/Integration/vRouterAgentService.Tests.ps1 [hunk 3]
```diff
@@ -12,56 +12,45 @@ Param (
...
-$TestConf = Get-TestConfiguration
 $Sessions = New-RemoteSessions -VMs (Read-TestbedsConfig -Path $TestenvConfFile)
 $Session = $Sessions[0]
 
 $ControllerConfig = Read-ControllerConfig -Path $TestenvConfFile
+$OpenStackConfig = Read-OpenStackConfig -Path $TestenvConfFile
+$SystemConfig = Read-SystemConfig -Path $TestenvConfFile
 
 Describe "vRouter Agent service" {
-    Context "enabling" {
```
Comment (katrybacka) [inline CIScripts/Test/Tests/Integration/vRouterAgentService.Tests.ps1]: Why is this removed? Maybe we should just mark this test as disabled?

File: CIScripts/Test/TestRunner.ps1 (hunk 2 above)
Comment (krdln) [reply CIScripts/Test/TestRunner.ps1]: It's converted to `-Pending` later on. So it's disabled, but shows up on report as inconclusive. See http://logs.opencontrail.org/winci/github/WinContrail/contrail-win-ci-gh/923/testReport.html

File: CIScripts/Test/Tests/Integration/vRouterAgentService.Tests.ps1 (hunk 3 above)
Comment (krdln) [reply CIScripts/Test/Tests/Integration/vRouterAgentService.Tests.ps1]: The test was identical to "runs correctly"

File: CIScripts/Test/TestConfigurationUtils.ps1 (hunk 1 above)
Comment (krdln) [reply CIScripts/Test/TestConfigurationUtils.ps1]: I'll remove this `TODO` – these lines will be removed in #44 anyway
